pyproject.toml

```diff
@@ -142,14 +142,15 @@ dodoFile = "dev.py"
 [tool.cibuildwheel]
 skip = "cp36-* cp37-* pp* *_ppc64le *_i686 *_s390x *-musllinux*"
 build-verbosity = "3"
-before-test = "pip install -r {project}/tools/wheels/test_requirements.txt"
+# gmpy2 and scikit-umfpack are usually added for testing. However, there are
+# currently wheels missing that make the test script fail.
+test-requires = ["pytest", "pytest-cov", "pytest-xdist", "asv", "mpmath", "threadpoolctl", "pooch"]
```

Comment:
For when we touch this file again: we can also remove `pytest-cov` and `asv` here, they aren't used.